skimage/util/arraycrop.py

```diff
@@ -38,13 +37,28 @@ def crop(ar, crop_width, copy=False, order='K'):
         The cropped array. If ``copy=False`` (default), this is a sliced
         view of the input array.
     """
-    ar = np.array(ar, copy=False)
-    crops = _as_pairs(crop_width, ar.ndim, as_index=True)
+    array = np.array(array, copy=False)
 
-    slices = tuple(slice(a, ar.shape[i] - b)
+    if isinstance(crop_width, int):
+        crops = [[crop_width, crop_width]] * array.ndim
+    elif isinstance(crop_width[0], int):
+        if len(crop_width) == 1:
+            crops = [[crop_width[0], crop_width[0]]] * array.ndim
+        elif len(crop_width) == 2:
+            crops = [crop_width] * array.ndim
+        else:
+            raise ValueError('invalid length for sequence crop_width')
```

Comment:
Maybe a message giving the length of `crop_width` and the expected ones will help?